### ISSUE TITLE
Print Episode Title instead of  generic title

### ISSUE THREAD

**Issue by danyboy666** [bvanheu/pytoutv#123]
For some reason when I pulled the last version the full episode title isn't displayed anymore.

`db666@homeshare:~$ toutv info "Les grands reportages" S24E02
Les grands reportages
La course vers l'intelligence artificielle [S24E02]

Ce documentaire australien démontre comment l'intelligence artificielle va
changer notre travail, car de nouvelles recherches montrent combien de ce que
nous faisons pourrait être fait par des robots.

Infos:

Air date: 2018-01-03
Available qualities:
461 kbps (480x270)
561 kbps (512x288)
925 kbps (640x360)
1324 kbps (852x480)
2124 kbps (852x480)
`
This is what the script pull with the lastest version:

`torrent@homeseedbox:~/script$ toutv info "Les grands reportages" S24E02
Les grands reportages
Épisode 02  [S24E02]

Ce documentaire australien démontre comment l'intelligence artificielle va
changer notre travail, car de nouvelles recherches montrent combien de ce que
nous faisons pourrait être fait par des robots.

Infos:

  * Air date: 3 janvier 2018
  * Available qualities:
    * 461 kbps (480x270)
    * 561 kbps (512x288)
    * 925 kbps (640x360)
    * 1324 kbps (852x480)
    * 2124 kbps (852x480)
    * 3124 kbps (1280x720)
`

I'm working on a script wich compare data from toutv and tvdb and automaticaly match and rewrite the tag with proper SXXEXX. Fetching the episode name directly from pytoutv and comparing that with the database would simplify alot of what i'm doing on my side :).

**Comment by simark** [comment]
There must be something that changed in the data we receive from the server, because I can't find the title at all in the data we receive from the server.  This is the request we do to get info on an episode:

https://github.com/bvanheu/pytoutv/blob/master/toutv/transport.py#L125

We probably need to re-check what requests the latest Android or iPhone application does, and see if there would be something more appropriate, with the information we want.

**Comment by danyboy666** [comment]
It's definitly the code I have to different versions of pytoutv running and they interpret the episode name differently as the exemple.

**Comment by simark** [comment]
Ok, interesting. Do you know which commit is the version that works from?  Or at least the version (`toutv --version`)?

**Comment by danyboy666** [comment]
Yeah I updated that version sadly. I know it's a really old version, one year+. I'll try to find a way to revert to that version or get the pull history if that's even possible. I know I installed that and never pulled the last commits untile sometime like last week or so.

**Comment by danyboy666** [comment]
`dbtech@DB-TECH-SRV0001:~/pytoutv$ toutv info  "Les grands reportages"
Les grands reportages  [Unknown country]

Les meilleurs documentaires d'ici et d'ailleurs sur des sujets d'actualité.


Infos:

  * Network: RDI
  * Removal date: 2014-08-11 00:00:00
dbtech@DB-TECH-SRV0001:~/pytoutv$ toutv info  "Les grands reportages" S24E02
Les grands reportages
La course vers l'intelligence artificielle  [S24E02]

Ce documentaire australien démontre comment l'intelligence artificielle va
changer notre travail, car de nouvelles recherches montrent combien de ce que
nous faisons pourrait être fait par des robots.


Infos:

  * Air date: 2018-01-03
  * Available qualities:
    * 461 kbps (480x270)
    * 561 kbps (512x288)
    * 925 kbps (640x360)
    * 1324 kbps (852x480)
    * 2124 kbps (852x480)
`

**Comment by danyboy666** [comment]
dbtech@DB-TECH-SRV0001:~/pytoutv$ toutv --version
toutv v2.3.0

**Comment by danyboy666** [comment]
it's version dating just a little over a year. 

So yeah v2 can return the proper episode title.

**Comment by danyboy666** [comment]
`dbtech@DB-TECH-SRV0001:~/pytoutv$ toutv login $user $password
usage: toutv [-h] [-n] [-v] [-V] {list,info,search,fetch,clean} ...
toutv: error: argument command: invalid choice: 'login' (choose from 'list', 'info', 'search', 'fetch', 'clean')
`

Damn I can't login to dl 720p content with this version :/

**Comment by danyboy666** [comment]
I think you might be right after all. It may have somthing to do with the data the server returns. I noticed in your version 2.3 that you call toutv api v1. If you call api v2 in the latest release the data might not be formatted the same.

**Comment by simark** [comment]
That's still a regression for the pytoutv end user, that doesn't care about what API we use.  It should be fixed (although I can't guarantee I'll be able to spend some time on it soon).

**Comment by danyboy666** [comment]
Sounds good. I can pull the episodes info
directly from the json data for now it'll will do the trick for me.



Sent from my Samsung Galaxy smartphone.


-------- Original message --------
